src/builders/transform/plugins.ts

```diff
@@ -0,0 +1,74 @@
+import consola from "consola";
+
+import type { Plugin } from "@obuild/plugin";
+
+export type PluginName =
+  | "oxc-transform"
+  | "oxc-minify"
+  | "oxc-dts"
+  | "tsc-dts"
+  | "tsgo-dts"
+  | "vue-sfc-transformer"
+  | "vue-tsc-dts"
+  | "sass"
+  | "postcss"
+  | (string & {});
+
+export type Plugins = Array<PluginName | Plugin<any>>;
+
+type PluginPackageName = string;
+type PluginDirectory = Record<PluginName, PluginPackageName | undefined>;
+
+// prettier-ignore
+const pluginDirectory: PluginDirectory = {
+  "oxc-transform":       "@obuild/plugin-oxc-transform",
+  "oxc-minify":          "@obuild/plugin-oxc-minify",
+  "oxc-dts":             "@obuild/plugin-oxc-dts",
+  "sass":                "@obuild/plugin-sass",
+  "postcss":             "@obuild/plugin-postcss",
+  "tsc-dts":             "@obuild/plugin-tsc-dts",
+  "tsgo-dts":            "@obuild/plugin-tsgo-dts",
+  "vue-sfc-transformer": "@obuild/plugin-vue-sfc-transformer",
+  "vue-tsc-dts":         "@obuild/plugin-vue-tsc-dts",
+};
```

Comment:
We can try `{name}` => `@obuild/plugin-${name}` instead of keeping a hardcoded list